apps/src/code-studio/components/progress/ScriptOverviewTopRow.jsx

```diff
@@ -11,6 +11,7 @@ import Assigned from '@cdo/apps/templates/Assigned';
 import {sectionForDropdownShape} from '@cdo/apps/templates/teacherDashboard/shapes';
 import {sectionsForDropdown} from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
 import TeacherResourcesDropdown from '@cdo/apps/code-studio/components/progress/TeacherResourcesDropdown';
+import UnitCalendarButton from './UnitCalendarButton';
```

Comment:
It would be great to use the `@cdo/...` path here because it means if we move things around they still work.